### ISSUE TITLE
Heap Implementation In C++

### ISSUE THREAD

**Issue by sobhanbera** [sobhanbera/algorithms#126]
#### This issue is about [Algo/DS Name/Question Name](link to resource for the Algo/DS/Question)

-   [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [ ] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [ ] I will send my PR only after I'm assigned to this issue by any maintainer

-   [ ] Assign this issue to me (I want to work on this)

-   [ ] I ensure that I am not already assigned to 2 or more issues

**Comment by Deepak2417** [comment]
Please assign this to me.

**Comment by Deepak2417** [comment]
Hey @SobhanBera ! Please clarify the language of the code C++ or Java?
As in issues tittle its C++ but in label it is Java.

I have already done with C++ code with proper comments. Till that I making PR for Heap Implementation in C++

**Comment by SobhanBera** [comment]
Sorry, Deepak
For late reply and Thanks For Collaborating in this repository.

**Comment by Deepak2417** [comment]
I have already made a PR #130 , you can review it for this issue.
